sdk/devcenter/developer-devcenter-rest/samples-dev/sampleCreateDevBox.ts

```diff
@@ -16,68 +15,36 @@ dotenv.config();
  */
 async function createDevBox() {
   // Build client and fetch required parameters
-  const endpoint = process.env.DEVCENTER_ENDPOINT || "<endpoint>";
-  const client = createClient(endpoint, new DefaultAzureCredential());
+  const devCenter = process.env.AZURE_DEVCENTER_NAME || "<devcenter name>";
```

Comment:
curious why this is changed to `AZURE_DEVCENTER_NAME`? The api seems to still take an endpoint?  The tests also pass in an endpoint.

Reply:
Reverted back to `DEVCENTER_ENDPOINT`